src/hooks/siteDashboardHooks/useGetSiteLaunchStatus.ts

```diff
@@ -16,6 +18,11 @@ export const useGetSiteLaunchStatus = (
     },
     {
       retry: false,
+      // we want to refetch the data every 5 minutes
+      // during the site launch process. Else, it will
+      // only be updated on a hard refresh.
+      refetchInterval: FIVE_MINUTES,
+      refetchIntervalInBackground: true,
```

Comment:
can i check that the component that calls this hook only exists when users attempt to launch their site? (iirc should be hor otherwise is the card alr)

Reply:
Uhm no not true, it also gets called when the site is in the dashboard. But since it is once every 5 mins I think this is ok, wdyt?

Reply:
Isn't that component conditionally shown and if no site launch then it's human image? That's off the top of my head but not v impt cos low freq

Reply:
yup this is accurate!